cmd/ocm-support/find/cmd.go

```diff
@@ -0,0 +1,22 @@
+package find
```

Comment:
are we keeping the command as **find**, not **get**?

Reply:
Missed it. Changed to **get**